sourcecode/www/src/i18n/en/translation.json

```diff
@@ -127,7 +127,7 @@
     "first_changed": "first changed",
     "most_used": "most used",
     "least_used": "least used",
-    "something_happened": "something happened",
+    "something_happened": "A server error occurred. Please try again later. If the problem persists, please contact the site administrator.",
```

Comment:
This message is repeated three times now. Please remove two of them and change the code to reference the one that remains.